microsoft/testsuites/mshv/mshv_root_tests.py

```diff
@@ -180,15 +183,19 @@ def _mshv_stress_vm_create(
             node.tools[Free].log_memory_stats_mb()
 
         for i in range(vm_count):
-            disk_img_file = node.working_path / f"VM{i}_{self.DISK_IMG_NAME}"
-            node.tools[Rm].remove_file(str(disk_img_file))
+            disk_img_file = disk_img_copy_path / f"VM{i}_{self.DISK_IMG_NAME}"
+            node.tools[Rm].remove_file(str(disk_img_file), sudo=True)
 
-        dmesg_str = node.tools[Dmesg].get_output()
-        dmesg_path = log_path / f"dmesg_{times}_{cpus_per_vm}_{mem_per_vm_mb}"
-        with open(str(dmesg_path), "w") as f:
-            f.write(dmesg_str)
         assert_that(failures).is_equal_to(0)
 
+    def _get_disk_img_copy_path(self, node: Node) -> PurePath:
+        # Azure temporary disk is mounted at /mnt. It has more space then OS
+        # disk. Use it for storing copies of the disk image if it exists.
+        if node.tools[Ls].path_exists("/mnt"):
```

Comment:
in some case, it is /mnt/resource

Reply:
This test can only run on our dom0 image. There it is /mnt. So, I deliberately hard coded it.

Reply:
If it doesn't run within Azure, add a platform requirement to skip Azure.

Reply:
It runs within Azure if you deploy the dom0 VHD.

Reply:
Is there an easy way to check if it's dom0 testing scenario? It helps to skip this test on other test runs.

Reply:
We are already doing a check in `before_suite` and skipping this test if it is not dom0. It is a very simple check: if /dev/mshv exists continue with test, else skip.